hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSet.java

```diff
@@ -84,7 +85,7 @@ public long get(Runtime runtime) {
                 new LongProbeFunction<Runtime>() {
                     @Override
                     public long get(Runtime runtime) {
-                        return runtime.availableProcessors();
+                        return RuntimeAvailableProcessors.get();
```

Comment:
I do not agree. This is the RuntimeMetricSet, so it will report what is in the Runtime. Not some kind of mocked value.